acme/tests/client_test.py

```diff
@@ -142,6 +144,15 @@ def test_query_registration_client_v2(self):
         self.response.json.return_value = self.regr.body.to_json()
         self.assertEqual(self.regr, client.query_registration(self.regr))
 
+    def test_query_registration_client_v2_ecdsa(self):
```

Comment:
I'm not really sure if this test is needed, or more importantly, what it's supposed to verify. I'm thinking it may be nice to check that you ACME client can post EC account keys?

Reply:
That's not a bad idea, but since that functionality wasn't added in this PR, I personally wouldn't worry too much about it here.